SimpleNtuplizer/test/runAnalyzer.py

```diff
@@ -54,12 +55,16 @@
     superClustersEB     = cms.InputTag("particleFlowSuperClusterECAL", "particleFlowSuperClusterECALBarrel"),
     superClustersEE     = cms.InputTag("particleFlowSuperClusterECAL", "particleFlowSuperClusterECALEndcapWithPreshower"),
     pfLabel             = cms.InputTag("particleFlowClusterECALMatchedToPhotons"),
+    pspfLabel             = cms.InputTag("particleFlowClusterECALMatchedToPhotons"),
```

Comment:
Ok, I drop my other comment. I think it's ok to leave them separately and just repeat the name.